### DISCUSSION TITLE
❔ Question: I'm using "Local" - is it using my storage?

### DISCUSSION THREAD

**Discussion by W1Real** [Seraphli/obsidian-link-embed#43] [Q&A] [answered]
Is the plugin only using bandwidth or does it store the thumbnails somewhere in my vault/storage? 

All it does in terms of storage is to fill in that information in the codeblock like this:
```embed
title: "photo-1562016600-ece13e8ba570 (4000×5298)"
image: "https://images.unsplash.com/photo-1562016600-ece13e8ba570?ixlib=rb-4.0.3"
description: ""
url: "https://images.unsplash.com/photo-1562016600-ece13e8ba570?ixlib=rb-4.0.3&q=85&fm=jpg&crop=entropy&cs=srgb&w=2400"
```
**So... It's only using bandwidth to load the thumbnail, right?**

**Comment by Seraphli** [comment, marked as answer]
The `Local` term in the setting means the plugin will get the meta data by opening the url in a hidden browser window and get the needed info. The image displayed in the plugin is get from the url, not from your local vault. But Obsidian have some cache mechanism so once the image is loaded, Obsidian will use the local cache when the plugin query for the image. After restart Obsidian, I guess the image will be dowloaded again. Clearly, when your vault is copied or synced to another computer, the image will be downloaded again.
